Makefile

```diff
@@ -15,7 +15,11 @@
 #    assistance with these files: Intel Corporation, Mellanox Technologies Ltd,
 #    Dell Products, L.P., Facebook, Inc
 #
-
+ifdef SAITHRIFTV2
```

Comment:
A comment here explaining why this is needed may help

Reply:
In addition, a CLI command on how to invoke this can be added here.

Reply:
thx. added more comment